app/routes/users/components/UserProfile.js

```diff
@@ -393,8 +393,10 @@ const UserProfile = (props: Props) => {
 
       <Flex wrap className={styles.header}>
         <div className={cx(styles.sidebar, styles.picture)}>
-          {hasFrame && <Image className={styles.frame} src={frame} />}
-          <ProfilePicture user={user} size={150} />
+          <div className={styles.picture}>
```

Comment:
I assume you use the `picture` class for its flex-box styling, but I think this should be switched to a `<Flex />` component instead. The class name does not convey what it does either.

Reply:
Yeah, sure. I switched to Flex now ✅

Reply:
I was thinking you could do it on the other div employing the same styling as well, and remove the `picture` class entirely.

Reply:
Sure. Removed the class now 🚀